README.md

```diff
@@ -1 +1,19 @@
-# hw01
\ No newline at end of file
+# MACS 20500: Homework 1  
+## James Zimmerman  
+
+  
+### Intro  
```

Comment:
Try to leave a blank line after headings in the future. It's not an issue here, but leaving space can help to prevent rendering issues in future assignments.